roles/ceph-common/tasks/main.yml

```diff
@@ -67,6 +78,12 @@
     is_ceph_infernalis={{ (ceph_stable and ceph_stable_release not in ceph_stable_releases) or (ceph_dev) or (ceph_stable_rh_storage and (rh_storage_version.stdout | version_compare('0.94', '>'))) }}
 
 - set_fact:
+    is_ceph_infernalis=True
```

Comment:
@leseb I was trying to avoid running a shell command to get the rhcs version on debian systems. In this case we can just assume that anything debian and rh storage is gonna be jewel or later.

I can make a change to get the rhcs_storage_version on debian systems if you'd like.